cmd/substation/test.go

```diff
@@ -211,7 +221,7 @@ func testFile(arg string, extVars map[string]string) error {
 
 		cfg = c
 	case ".json":
-		c, err := fiConfig(arg)
+		c, err := fiConfig(fileName)
 		if err != nil {
 			return err
 		}
```

Comment:
No, these commands require a different behavior than the read command. These intentionally skip unhandled file types so that testing and vetting isn't disrupted, and the "default case" is implied (it's a no-op). The default case exists in the read command for ease of use.